lib/processLayerStyles.js

```diff
@@ -1,17 +1,25 @@
-const { addOptionalProperty, colorToRGBA, doRound } = require('./processShared')
+const { addOptionalProperty, colorToRGBA, doRound, prepareAssetBundleForPDF, prepareAssetBundleForPNG } = require('./processShared')
+const fs = require("fs")
 
+const FILL_TYPE = {
+  SOLID_COLOR: 0,
+  GRADIENT: 1,
+  IMAGE: 4,
+}
```

Comment:
ha ha this has bit mask written all over it - what happened to 2? Also, not sure why they've chosen a bit mask given you can't really combine them!